release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+of OpenShift engineering. Red Hat continues to influence the product in the
+areas of storage, networking, resource management, authentication and
+authorization, multi-tenancy, security, service deployments, templating, and
+controller functionality.
+
+[[ocp-37-crio]]
+==== CRI-O (Technology Preview)
+
+CRI-O v1.0 (currently in xref:ocp-37-technology-preview[Technology Preview]) is a
+lightweight, native Kubernetes container runtime interface. By design, it
+provides only the runtime capabilities needed by the kublet. CRI-O is designed
+to be part of Kubernetes and evolve in lock-step with the platform.
+
+CRI-O brings:
+
+* A minimal and secure architecture.
+* Excellent scale and performance.
+* The ability to run any Open Container Initiative (OCI) or docker image.
+* Familiar operational tooling and commands.
+
+image::crio-3-7.png[CRI-O]
+
+[[ocp-37-cluster-wide-tolerations-per-namespace-tolerations]]
+==== Cluster-wide Tolerations and Per-namespace Tolerations to Control Pod Placement
+
+In a multi-tenant environment, you want to leverage administration controllers
+to help define rules that can help govern a cluster, should a tenant not set a
+toleration for placement.
+
+The following is offered to administrators where the namespace setting will
+override the cluster setting:
+
+* Cluster-wide and per-namespace default toleration for pods.
+* Cluster-wide and per-namespace white-listing of toleration for pods.
+
+.Cluster-wide Off Example
+----
+admissionConfig:
+  pluginConfig:
+    PodTolerationRestriction:
+      configuration:
+        kind: DefaultAdmissionConfig
+        apiVersion: v1
+        disable: true
+----
+
+.Cluster-wide On Example
+----
+admissionConfig:
+  pluginConfig:
+    PodTolerationRestriction:
+      configuration:
+        apiVersion: podtolerationrestriction.admission.k8s.io/v1alpha1
+        kind: Configuration
+        default:
+         - key: key3
+           value: value3
+        whitelist:
+         - key: key1
+           value: value1
+         - key: key3
+           value: value3
+----
+
+.Namespece-specific Example
+----
+piVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: ""
+    openshift.io/display-name: ""
+    openshift.io/sa.scc.mcs: s0:c8,c7
+    openshift.io/sa.scc.supplemental-groups: 1000070000/10000
+    openshift.io/sa.scc.uid-range: 1000070000/10000
+    scheduler.alpha.kubernetes.io/defaultTolerations: '[ { "key": "key1", "value":"value1" }]'
+    scheduler.alpha.kubernetes.io/tolerationsWhitelist: '[ { "key": "key1", "value":
+      "value1" }, { "key": "key2", "value": "value2" } ]'
+  generateName: dma-
+spec:
+  finalizers:
+  - openshift.io/origin
+  - kubernetes
+----
+
+[[ocp-37-security]]
+=== Security
+
+[[ocp-37-documented-private-public-key-configurations-and-crypto-levels]]
+==== Documented Private and Public Key Configurations and Crypto Levels
+
+{product-title} is a secured by default implementation of Kubernetes.
```

Comment:
The following is confusing (I know what you're trying to say, it just does not say it - cleanly). The statement also does not connect to what this section is trying to say. 
 
Try: While OpenShift Container Platform is a secured by default implementation of Kubernetes, we are now publicly documenting what security protocols and ciphers we use.